tests/validators.py

```diff
@@ -194,6 +194,31 @@ def validate_registry():
     kubectl("delete -f {}".format(manifest))
 
 
+def validate_registry_custom(size, storageclass):
+    """
+    Validate the private registry with custom size and storageclass
+    """
+
+    wait_for_pod_state("", "container-registry", "running", label="app=registry")
+    pvc_stdout = kubectl("get pvc registry-claim -n container-registry -o yaml")
+    pvc_yaml = yaml.safe_load(pvc_stdout)
+    storage = pvc_yaml["spec"]["resources"]["requests"]["storage"]
+    storageClassName = pvc_yaml["spec"]["storageClassName"]
+    assert storage == size
+    assert storageClassName == storageclass
+    assert re.match("(^[2-9][0-9]{1,}|^[1-9][0-9]{2,})(Gi$)", storage)
```

Comment:
nit; i would remove this, this assertion is beside the point of the test (which is to ensure that the correct size and storage class was used).